antarest/study/storage/variantstudy/variant_study_service.py

```diff
@@ -481,18 +621,25 @@ def get_raw(self, metadata: VariantStudy) -> FileStudy:
             metadata: study
         Returns: the config and study tree object
         """
-        raise NotImplementedError()
+        self._safe_generation(metadata)
+
+        study_path = self.get_study_path(metadata)
+        study_config, study_tree = self.study_factory.create_from_fs(
+            study_path, metadata.id
+        )
+        return FileStudy(config=study_config, tree=study_tree)
 
     def get_study_sim_result(
-        self, metadata: VariantStudy
+        self, study: VariantStudy
     ) -> List[StudySimResultDTO]:
         """
         Get global result information
         Args:
             study: study
         Returns: study output data
         """
-        raise NotImplementedError()
+        self._safe_generation(study)
+        return super().get_study_sim_result(study=study)
```

Comment:
this is specific to variant studies since the output directory is not located within the study file tree